### ISSUE TITLE
feature request: import/export settings

### ISSUE THREAD

**Issue by deejay-girl** [mixxxdj/mixxx#12127]
### Feature Description

I know there is mixxx.cfg but in order to have a user friendly way to transfer settings between computers, a feature could be made to import/export settings. In this way the user don't have to copy the file manually to the right folder and let Mixxx take care of it. 

See screenshot of how Traktor is doing it. 

![afbeelding](https://github.com/mixxxdj/mixxx/assets/147563220/4cf5107d-aa24-4c2b-9664-41a2b45da052)

- import button: loads a mixxx.cfg file 

- export button: saves a mixxx.cfg file based on the current settings. Traktor also allows to choose what to export (this is maybe not necessary because everything could be exported to the file)

![afbeelding](https://github.com/mixxxdj/mixxx/assets/147563220/d19db5e0-d7a2-4fcf-93d2-0aacdbdb4d09)

Those 2 buttons could be added to the settings window of Mixxx.

**Comment by daschuer** [comment]
> The goal is to make the destination computer have the identical settings of the source computer in a user friendly way.

Such a big list of preferences option is not user friendly. 
What is the use case for each option? 

You can already put your .mixxx folder along with your audio files on a shared drive and start Mixxx with --settings-path 

The issue with it is that the external drive might slow down the SQLite and puts the database file on the risk when unplugged during access. 

A feature that works around this would be welcome.

**Comment by xxxmicrobexxx** [comment]
I agree that this feature would be useful.  When changing settings it is useful to have an easily accessible backup
